sysutils/ipbt/Portfile

```diff
@@ -33,6 +38,9 @@ depends_lib         port:ncurses
 
 cmake.build_type    Debug
 
+# https://trac.macports.org/ticket/66737
+compiler.blacklist-append *gcc-4.* {clang < 400}
```

Comment:
Why not use `compiler.c_standard`?

Reply:
@jmroot It does not solve the issue: https://trac.macports.org/ticket/66737#comment:2

Reply:
I'm not suggesting using `configure.cflags-append -std=c99`, but rather `compiler.c_standard 2011`.

Reply:
Does it require C++11 anywhere? I could have missed that.

Reply:
It doesn't require C++ at all as far as I can see.

Reply:
> It doesn't require C++ at all as far as I can see.

Sorry, I meant C11, of course.

Reply:
As I mentioned in the ticket, it is using a C11 feature, yes.

Reply:
Should I make a PR to change blacklist to c_standard?